.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "javascriptreact",
     "typescript",
     "typescriptreact"
-  ]
+  ],
+  "vitest.disableWorkspaceWarning": true
```

Comment:
Would be great to address this as some point. Previously flagged [here](https://hashintel.slack.com/archives/C03TWBSRT8B/p1718695843240629) (_internal link_).